### ISSUE TITLE
Can not transition properties of objects created with from and wrapped in an Identity

### ISSUE THREAD

**Issue by taras** [microstates/microstates.js#283]
I would expect the following to pass but it fails.

```js
  describe('identity support for microstates created with from(object)', () => {
    let i;
    beforeEach(() => {
      i = Identity(from({ name: 'Taras' }), next => i = next);
      i.name.concat('!!!');
    });
    it('allows to transition value of a property', () => {
      expect(i.name.state).toBe('Taras!!!');
    });
  });
```

This test is in [tm/destructuring-from-object](https://github.com/microstates/microstates.js/tree/tm/destructuring-from-object) branch.

**Comment by taras** [comment]
One observation is that properties on the object are microstates, not identities.

**Comment by cowboyd** [comment]
Is this resolved by #284 ?
